libpod/info.go

```diff
@@ -178,3 +186,26 @@ func (r *Runtime) GetOCIRuntimeVersion() (string, error) {
 	}
 	return strings.TrimSuffix(output, "\n"), nil
 }
+
+// GetHostDistributionInfo returns a map containing the host's distribution and version
+func (r *Runtime) GetHostDistributionInfo() map[string]string {
+	dist := make(map[string]string)
+
+	f, err := os.Open("/etc/os-release")
+	defer f.Close()
+	if err != nil {
+		return dist
+	}
+
+	l := bufio.NewScanner(f)
+
+	for l.Scan() {
+		if strings.HasPrefix(l.Text(), "ID=") {
+			dist["Distribution"] = strings.Replace(strings.TrimSpace(strings.Join(strings.Split(l.Text(), "=")[1:], "")), "\"", "", -1)
```

Comment:
Use TrimPrefix() instead of the Split/Join here

Reply:
Probably don't need the replace either

Reply:
ok, done